gui/slick/views/snatchSelection.mako

```diff
@@ -98,7 +103,7 @@
                 % endif
             </ul>
         </div><!-- #tags //-->
-        <div id="summary">
+        <div id="summary" ${"class=\"summaryFanArt\"" if bool(sickbeard.FANART_BACKGROUND) else ""}>
```

Comment:
@Thraxis need to remove all the `bool`. apart from that is there anything else that needs to be done?